bottom-rs.mk

```diff
@@ -0,0 +1,39 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS       += bottom-rs
+BOTTOM-RS_COMMIT  := 3451cdadd7c4e64fe8e7f43e986a18628a741dec
+BOTTOM-RS_VERSION := 1.0+git20210404.$(shell echo $(BOTTOM-RS_COMMIT) | cut -c -7)
+DEB_BOTTOM-RS_V   := $(BOTTOM-RS_VERSION)
+
+bottom-rs-setup: setup
+	$(call GITHUB_ARCHIVE,bottom-software-foundation,bottom-rs,$(BOTTOM-RS_COMMIT),$(BOTTOM-RS_COMMIT))
+	$(call EXTRACT_TAR,need_top.tar.gz,bottom-rs-need_top,bottom-rs)
+
+ifneq ($(wildcard $(BUILD_WORK)/bottom-rs/.build_complete),)
+bottom-rs:
+	@echo "Using previously built bottom-rs."
+else
+bottom-rs: bottom-rs-setup
+	cd $(BUILD_WORK)/bottom-rs/ && cargo install --target $(RUST_TARGET) --root $(BUILD_STAGE)/bottom-rs$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/ --path .
+	touch $(BUILD_WORK)/bottom-rs/.build_complete
+endif
+
+bottom-rs-package: bottom-rs-stage
+	# bottom-rs.mk Package Structure
+	rm -rf $(BUILD_DIST)/bottom-rs
+	mkdir -p $(BUILD_DIST)/bottom-rs
```

Comment:
You should get rid of rust/cargo-generated files in here, which are added to ``MEMO_SUB_PREFIX/bin``. This should prevent from extra files being installed alongside other bins.